core/wallet/did.go

```diff
@@ -72,17 +73,41 @@ func (w *Wallet) IsDIDExist(did string) bool {
 }
 
 func (w *Wallet) AddDIDPeerMap(did string, peerID string) error {
```

Comment:
How do we add `did_last_char` to `DIDPeerTable`  of existing DIDs and publish to network?

Reply:
Please refer the 'func (w *Wallet) AddDIDLastChar()' , this function will check for DIDs with did_last_char column as null and update them with corresponding last character.